admin/common_auth/views.py

```diff
@@ -21,6 +24,47 @@ def login(request):
     context = {'form': form}
     return render(request, 'login.html', context)
 
+
 def logout(request):
     logout_user(request)
     return redirect('auth:login')
+
+
+# Permission restriction based on staff (only staff can register new users). Use is_staff boolean?
+def register(request):
+    if request.user.is_staff:
+        if request.method == 'POST':
+            form = CustomUserRegistrationForm(request.POST)
+            if form.is_valid():
+                email = form.cleaned_data['email']
+                password = form.cleaned_data['password1']
```

Comment:
I checked to make sure that password confirmation is at least being performed on the client side.

![screen shot 2016-03-21 at 1 45 02 pm](https://cloud.githubusercontent.com/assets/134455/13928200/224ca950-ef6b-11e5-8822-3ac87a0321ba.png)